lib/rails_admin/config.rb

```diff
@@ -254,6 +257,7 @@ def models
       # @see RailsAdmin::Config.registry
       def reset
         @compact_show_view = true
+        @browser_validations = false
```

Comment:
@mshibuya If you prefer I can set the default value to true. It's not a problem for me. 

My personal opinion is that browser validation is a nice to have feature only for simple application's models.

In my experience there are two use cases that it could be a trouble:

- When a model has has_many/has_one nested attributes to be validated the browser-validation is generating a lot of js errors as commented above. 

- When a model has conditional validations.


Let me know what do you prefer as default value so I can do a commit and rebase the PR :wink: